client-ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -345,20 +345,14 @@ describe("hubConnection", () => {
             });
 
             it("closed with error if hub cannot be created", (done) => {
-                const errorRegex = {
-                    LongPolling: "Internal Server Error",
-                    ServerSentEvents: "Error occurred",
-                    WebSockets: "1011|1005", // Message is browser specific (e.g. 'Websocket closed with status code: 1011'), Edge and IE report 1005 even though the server sent 1011
-                };
-
                 const hubConnection = new HubConnection("http://" + document.location.host + "/uncreatable", {
                     logger: LogLevel.Trace,
                     protocol,
                     transport: transportType,
                 });
 
                 hubConnection.onclose((error) => {
-                    expect(error.message).toMatch(errorRegex[TransportType[transportType]]);
+                    expect(error.message).toEqual("Server returned an error on close: Connection closed with an error. InvalidOperationException: Unable to resolve service for type 'System.Object' while attempting to activate 'FunctionalTests.UncreatableHub'.");
```

Comment:
:tada:

Reply:
Don't celebrate too soon. This works on my computer but on the build server the message is sometimes the old error. I'll need to talk to you about it.